### PR TITLE
chore: move smart edit launch flag to smart-edit.service.ts

### PR DESCRIPTION
**Pull request by znikola**
This PR moves the `_launchInSmartEdit` flag to `smart-edit.service.ts`.

### CODE REVIEW

File: projects/core/src/cms/model/content-slot-component-data.model.ts [hunk 1]
```diff
@@ -2,5 +2,5 @@ export interface ContentSlotComponentData {
   uid?: string;
   typeCode?: string;
   flexType?: string;
-  properties?: any;
```
Comment (dunqan) [inline projects/core/src/cms/model/content-slot-component-data.model.ts]: I would reconsider this change, as `object` here doesn't add much in terms of type safety, but make using it harder.

File: projects/core/src/cms/model/content-slot-data.model.ts [hunk 2]
```diff
@@ -2,5 +2,5 @@ import { ContentSlotComponentData } from './content-slot-component-data.model';
 
 export interface ContentSlotData {
   components?: ContentSlotComponentData[];
-  properties?: any;
```
Comment (dunqan) [inline projects/core/src/cms/model/content-slot-data.model.ts]: I would reconsider this change, as object here doesn't add much in terms of type safety, but make using it harder.

File: projects/core/src/cms/model/content-slot-component-data.model.ts (hunk 1 above)
Comment (znikola) [reply projects/core/src/cms/model/content-slot-component-data.model.ts]: Done

File: projects/core/src/cms/model/content-slot-data.model.ts (hunk 2 above)
Comment (znikola) [reply projects/core/src/cms/model/content-slot-data.model.ts]: Done

**Review by dunqan — APPROVED** [review]

**Pull request merged**